Sail/src/Sail/graphics/geometry/Transform.h

```diff
@@ -83,7 +83,11 @@ class Transform {
 	glm::vec3& getForward();
 
 	// Matrix used by collision etc.
-	glm::mat4 getMatrix();
+	void prepareMatrix();
+
+	glm::mat4 getMatrixWithUpdate();
+
+	glm::mat4 getMatrixWithoutUpdate() const;
 
 	// Matrix used to render
 	glm::mat4 getRenderMatrix(float alpha = 1.0f);
```

Comment:
should these three methods really return a mat4 copy or can they be changed to `const glm::mat4&` ?

Reply:
has been changed to `const glm::mat4` now